CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 ## Version 0.2.0
 
+* Allow to select location to save burndown chart image through `--location`
```

Comment:
this should have been in the _Master (unreleased)_ section, but we have decided in https://github.com/openSUSE/trollolo/pull/163 to stop using this. So please remove this, it is not needed any more. And sorry for the mess. 😄